source/runner/Executor.ts

```diff
@@ -84,6 +84,7 @@ export class Executor {
         console.log("Found some message, writing it down")
       }
       const comment = githubResultsTemplate(results)
+      // TODO filter out Danger items that contain file/line info
```

Comment:
Yes, this is where you want to find:

* Things with no file/line (main comment)
* Things with file/line inside the diff (inline)
* Things with file/line outside the diff (main comment)